### PR TITLE
enhancement: Don't show trackers for uninstalled apps

### PR DESCRIPTION
**Pull request by SamantaTarun**
Currently apps are not being removed from the list after they have been uninstalled.
Closes:  #302 
@Jean-BaptisteC

### CODE REVIEW

**Comment by tarunsamanta2k20** [comment]
> Why not reuse validPackages variable initialised each time refresh in launchFetch function in fetchUninstalledApps to get app list to reduce calls ? And please remove unused semicolons

I have compared the validPackages and then I will be able to say that the app has been deleted.

**Comment by tarunsamanta2k20** [comment]
@Jean-BaptisteC does exodusapp.packagename and application.packagename are both the same?

**Comment by Jean-BaptisteC** [comment]
Yes
Your condition cannot be work because each time validPackages and validPackagesafterUnistalled have same size because you call same function

**Comment by tarunsamanta2k20** [comment]
> Yes and why not create a new DAO to get only packagename ?

Please check the recent commit.

File: app/src/main/java/org/eu/exodus_privacy/exodusprivacy/ExodusUpdateService.kt [hunk 1]
```diff
@@ -358,6 +362,31 @@ class ExodusUpdateService : LifecycleService() {
...
+//        if(apptobeRemove.isNotEmpty()){
+//            val listOfPackageName = mutableListOf<String>();
+//            apptobeRemove.forEach{ listOfPackageName.add(it.packageName) }
+//            exodusDatabaseRepository.deleteApps(listOfPackageName)
+//        }
+        applicationListAfterUninstalled=exodusPackageRepository.getApplicationList(validPackages)
+
+        var packageNameList= listOf<String>();
+        packageNameList=exodusDatabaseRepository.getAllPackageNames();
+
```
[the remark this answers is not in the document]
Comment (Jean-BaptisteC) [reply app/src/main/java/org/eu/exodus_privacy/exodusprivacy/ExodusUpdateService.kt:385]: Yes if packageNameList.size > applicationListAfterUninstalled.size then remove uninstalled apps

**Comment by Jean-BaptisteC** [comment]
Works on emulator, apps uninstalled or disabled are remove from database
Please fix errors in linter

**Review by Jean-BaptisteC — APPROVED** [review]
✅ Pixel 6 - Android 13
✅ Instrumentation tests

**Pull request merged**